plugins/out_s3/s3.c

```diff
@@ -120,8 +119,9 @@ static char *mock_error_response(char *error_env_var)
     return NULL;
 }
 
-/* static void s3_retry_warn (struct flb_s3 *ctx, struct s3_file *chunk, char create_time[80], struct multipart_upload *m_upload){
-    strftime(create_time, 80, "%Y-%m-%d %H:%M:%S", localtime(&(chunk->create_time)));
+static void s3_retry_warn (struct flb_s3 *ctx, struct s3_file *chunk, char create_time[20], struct multipart_upload *m_upload){
+    struct tm now_time;
+    strftime(create_time, 20, "%Y-%m-%d %H:%M:%S", localtime_r(&(chunk->create_time), &now_time));
```

Comment:
why 20 characters?

Reply:
Because the fix length of the time is 20